client/src/lib/graphql/queries.js

```diff
@@ -23,6 +46,39 @@ export const GET_BOARDS = gql`
 	}
 `;
 
+export const GET_COLUMN_BY_ID = gql`
+	query GetColumnById($boardId: ID!, $columnId: ID!) {
+		column(boardId: $boardId, columnId: $columnId) {
+			id
+			name
+			tasks {
+				id
+				title
+				description
+				status
+				subtasks {
+					title
+					isCompleted
+				}
+			}
+		}
+	}
+`;
+
+export const GET_TASK_BY_ID = gql`
+	query GetTaskById($columnId: ID!, $taskId: ID!) {
+		task(columnId: $columnId, taskId: $taskId) {
+			id
+			title
+			description
+			status
+			subtasks {
+				title
+				isCompleted
+			}
+		}
+	}
+`;
```

Comment:
I think this is fine, but we should perhaps use a Schema.Graphql and save all these in it? We could refactor if we have time.